website/views/shared/tasks/edit/habits/plus_minus.jade

```diff
@@ -1,8 +1,8 @@
-fieldset.option-group.plusminus(ng-if='task.type=="habit" && canEdit(task)')
+fieldset.option-group.plusminus(ng-if='task.type === "habit" && canEdit(task)')
   legend.option-title=env.t('direction/Actions')
   span.task-checker
-    input.visuallyhidden.focusable(id='{{obj._id}}_{{task._id}}-option-plus', type='checkbox', ng-model='task.up')
-    label(for='{{obj._id}}_{{task._id}}-option-plus')
+    input.visuallyhidden.focusable(id='{{obj._id}}_{{task._edit._id}}-option-plus', type='checkbox', ng-model='task._edit.up')
```

Comment:
All these ids can't be changed, so they should reference `task._id`, not `task._edit._id`